Marlin/language_cz.h

```diff
@@ -58,11 +61,13 @@
 #define MSG_PREHEAT_1                       _UxGT("Zahrat PLA")
 #define MSG_PREHEAT_1_N                     MSG_PREHEAT_1 _UxGT(" ")
 #define MSG_PREHEAT_1_ALL                   MSG_PREHEAT_1 _UxGT(" Vse")
+#define MSG_PREHEAT_1_END                   MSG_PREHEAT_1 _UxGT(" Tryska")
 #define MSG_PREHEAT_1_BEDONLY               MSG_PREHEAT_1 _UxGT(" Podloz")
 #define MSG_PREHEAT_1_SETTINGS              MSG_PREHEAT_1 _UxGT(" Nast")
 #define MSG_PREHEAT_2                       _UxGT("Zahrat ABS")
 #define MSG_PREHEAT_2_N                     MSG_PREHEAT_2 _UxGT(" ")
 #define MSG_PREHEAT_2_ALL                   MSG_PREHEAT_2 _UxGT(" Vse")
+#define MSG_PREHEAT_2_END                   MSG_PREHEAT_2 _UxGT(" Tryska")
```

Comment:
MSG_PREHEAT_2_END should be translated to "End" or "finished".
I guess "Tryska" means "Nozzle"